content/sensu-go/5.16/guides/using-check-aggregates.md

```diff
@@ -0,0 +1,285 @@
+---
+title: ""
+linkTitle: "Using Check Aggregates"
+description: ""
+weight: 350
+version: "5.16"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-5.16:
+    parent: guides
+---
+
+Sensu provides the capability to monitor groups of checks or entities via aggregates. In Sensu Go, this is done through the use of labels. In this tutorial, we'll walk through configuring both an entity aggregate and a check aggregate.
+
+# Entity Aggregate
+
+Let's start with configuring an entity.
+
+The first thing we'll need to do is add a label to our agent.
+
+```yaml
+---
+# Sensu agent configuration
+
+##
+# agent overview
+##
+
+labels:
+  server_type: "webservers"
+
+```
+
+In this example, we've added the label `server_type` with the value `"webservers"`--it's useful to know if a number of agents on our webservers stop reporting in rather than just a single node. Let's add a similar label to a check.
+
+# Check Aggregate
+
+The check we'll use for our aggregate is an http metric check for a ficticious app site 
+
+```yaml
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: cpu-check
+  namespace: default
+  labels:
+    app_group: "identity_servers"
+spec:
+  command: "check-http.rb -u https://identity.{{ .entity.name }}.example.com"
+  interval: 10
+  publish: true
+  handlers:
+  - slack
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+
+```
+
+While Sensu is capable of monitoring singular entities, it is also capable of monitoring groups of entities (and checks) through the use of aggregates. _______________ . Let's get started.
+
+===
+
+## Entity Aggregates
+
+In order for a check or entity to be considered as part of an aggregate it must have a label, or set of labels assigned to it. We'll start by setting up a fictional scenario in which we have 20 webservers serving a number of applications. In this scenario, we might not care if a singular server stops responding, but we might care if say, 15 of the 20 stop responding. 
+
+To make an entity part of an aggregate, we would need to add a label in our `/etc/sensu/agent.yml`. For example:
```

Comment:
```suggestion
Add a label in your `/etc/sensu/agent.yml`:
```

Reply:
This may be confusing because it doesn't explain why we're only adding a label for one agent and not 20.